spec/system/task_spec.rb

```diff
@@ -1,27 +1,37 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { FactoryBot.create(:project) }
   describe 'Task一覧' do
+    let!(:task) { FactoryBot.create(:task, project_id: project.id) }
```

Comment:
`Factory`にassociationを記載することで`project_id`を省略して定義できますね。
```suggestion
    let!(:task) { FactoryBot.create(:task) }
```

Reply:
e551a09 でassociationを追記して`project_id`を省略しました。